agent/command/s3_put.go

```diff
@@ -509,13 +509,14 @@ func (s3pc *s3put) attachFiles(ctx context.Context, comm client.Communicator, lo
 		}
 
 		files = append(files, &artifact.File{
-			Name:       displayName,
-			Link:       fileLink,
-			Visibility: s3pc.Visibility,
-			AwsKey:     key,
-			AwsSecret:  secret,
-			Bucket:     bucket,
-			FileKey:    fileKey,
+			Name:        displayName,
+			Link:        fileLink,
+			Visibility:  s3pc.Visibility,
+			AwsKey:      key,
+			AwsSecret:   secret,
+			Bucket:      bucket,
+			FileKey:     fileKey,
+			ContentType: s3pc.ContentType,
```

Comment:
Can we add a test for this? There's already [a test that inspects the artifacts that would be sent to the app server](https://github.com/evergreen-ci/evergreen/blob/d7daa79cf9adb8145cc0bbe23905bb320d05c150/agent/command/s3_put_test.go#L342).

Reply:
Thanks done!